src/dsp_tools/commands/project/create/project_create.py

```diff
@@ -51,23 +49,13 @@ def _create_project_on_server(
     """
     all_projects = Project.getAllProjects(con=con)
     if project_definition.shortcode in [proj.shortcode for proj in all_projects]:
-        project_local = Project(con=con, shortcode=project_definition.shortcode)
-        project_remote: Project = project_local.read()
         msg = (
-            f"A project with shortcode {project_definition.shortcode} already exists on the DSP server. Updating it..."
```

Comment:
@jnussbaum Last week we decided to raise an error here, but Balduin pointed out that the rest of the program also updates lists, ontologies, etc, which is a feature we want. Therefore I changed it back to a message.

Reply:
I'm not sure if this is a feature we want. I rather see it this way: If a user executes `dsp-tools create xyz.json`, and the JSON file contains some info that is already on the server (e.g. a list that is partially on the server, but some new nodes will be updated), then this should be an error. I dislike all this updating stuff.

Reply:
That shouldn't happen, but it is possible to add a completely new list:
https://github.com/dasch-swiss/dsp-tools/blob/8c667a69d981d4dff0ca2f1fc3f8534c95d93ca0/src/dsp_tools/commands/project/create/project_create_lists.py#L120

Reply:
I also don't like the idea of updating anything. But as long as we don't have any way of creating an ontology without the `create` command, I find it crucial to be able to add ontologies to existing projects. And adding an ontology to a project is not the same as updating a project.